tests/TestUTestMacro.cpp

```diff
@@ -359,6 +359,30 @@ IGNORE_TEST(UnitTestMacros, UNSIGNED_LONGLONGS_EQUAL_TEXTWorksInAnIgnoredTest)
     UNSIGNED_LONGLONGS_EQUAL_TEXT(1, 0, "Failed because it failed"); // LCOV_EXCL_LINE
 } // LCOV_EXCL_LINE
 
+#else
+
+static void _LONGLONGS_EQUALFailsWithUnsupportedFeatureTestMethod()
+{
+    LONGLONGS_EQUAL(1, 1);
```

Comment:
@basvodde ummm, yeah, but it could provoke a compiler error if (by mistake) it gets passed to code that only takes unsigned. I agree about -1 being used as a special value, so -17 perhaps :-) ?